www/package.json

```diff
@@ -56,7 +56,7 @@
   "devDependencies": {
     "babel-eslint": "^10.0.1",
     "css": "^3.0.0",
-    "eslint": "^7.6.0",
```

Comment:
this one also cause warnings about peer dependencies, to be honest not sure why we need separate `eslint` for docs site..

Reply:
Ha, yeah... I guess `www` could have its own eslint config defined for the folder, but could still inherit from the parent `eslint` from the root `node_modules`.